src/helper/message.js

```diff
@@ -119,15 +119,29 @@ function postEMail(config, mail, callback) {
           'text': markdownText
         }
       }).done(function (response) {
-        callback(response);
+        return sendToLog(config.server.replace(/.*\/\/([^.]+).*/, '$1'), config.userId, response.message.rid).done(function () {
+          callback(response);
+        })
       }).fail(function (error) {
         callback(null, error);
       });
     }
   });
 
-
-
+  function sendToLog(env, userId, parent) {
```

Comment:
Can we make the request generic? Instead of using the pre-defined url would limit the plugin.

Reply:
Would be great, but I don't have an idea how...
How could the GitHub-Pages based "installation" get a logger URL injected?

Reply:
@ThomasRoehl mentioned we should use a bit.ly link